src/cnxml/index.js

```diff
@@ -32,18 +33,30 @@ export function writeXml(content, options={}) {
         >
         <title>TODO: load and preserve titles</title>
         <content>
-            {content}
+            {document}
         </content>
+        {
+            glossary ?
+                <glossary>
+                    {glossary}
+                </glossary>
+            : ''
+        }
```

Comment:
I'd prefer this check outside of JSX. Normal ifs are much easier to read.

Reply:
You mean something like this:
```
let r
    
    if (glossary) {
        r = <document
            xmlns="http://cnx.rice.edu/cnxml"
            cnxml-version="0.7"
            id="new"
            module-id="new"
            xmlLang="en"
            >
            <title>TODO: load and preserve titles</title>
            <content>
                {document}
            </content>
            <glossary>
                {glossary}
            </glossary>
        </document>
    } else {
        r = <document
            xmlns="http://cnx.rice.edu/cnxml"
            cnxml-version="0.7"
            id="new"
            module-id="new"
            xmlLang="en"
            >
            <title>TODO: load and preserve titles</title>
            <content>
                {document}
            </content>
        </document>
    }
```